packages/web3-test/src/index.ts

```diff
@@ -67,6 +76,52 @@ export async function testNodeWallet(baseUrl = 'http://127.0.0.1:22973'): Promis
   return wallet
 }
 
+function tryGetDevnetNodeProvider(): NodeProvider {
+  try {
+    return web3.getCurrentNodeProvider()
+  } catch (err) {
+    const nodeProvider = new NodeProvider('http://127.0.0.1:22973')
+    web3.setCurrentNodeProvider(nodeProvider)
+    return nodeProvider
+  }
+}
+
+export async function getSigner(alphAmount = ONE_ALPH * 100n): Promise<PrivateKeyWallet> {
```

Comment:
We could make signer group an option `getSigner(alphAmount = ONE_ALPH * 100n, group = 0)`. The default group is `rootWallet.group = 0`.